ieee.csl

```diff
@@ -214,20 +225,27 @@
         <text variable="DOI" prefix="doi: "/>
       </else-if>
       <else>
```

Comment:
given how we're using this now, you'll want to turn this into `<else-if variable="URL">`

Reply:
Done.